test/tstDBSCAN.cpp

```diff
@@ -74,27 +74,29 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(dbscan_verifier, DeviceType, ARBORX_DEVICE_TYPES)
     auto points = toView<DeviceType, Point>(
         {{{0, 0, 0}}, {{1, 1, 1}}, {{3, 3, 3}}, {{6, 6, 6}}});
 
-    auto r = std::sqrt(3);
+    auto r = std::sqrt(3.f);
+    auto r2 = std::sqrt(12.f);
+    auto r3 = std::sqrt(48.f);
 
     BOOST_TEST(verifyDBSCAN(space, points, r, 2,
                             toView<DeviceType, int>({1, 1, -1, -1})));
     BOOST_TEST(verifyDBSCAN(space, points, r, 3,
                             toView<DeviceType, int>({-1, -1, -1, -1})));
 
-    BOOST_TEST(verifyDBSCAN(space, points, 2 * r, 2,
+    BOOST_TEST(verifyDBSCAN(space, points, r2, 2,
```

Comment:
`2*r` with the new definition as a float is not good enough?

Reply:
No.